main/src/main/java/org/mobilitydata/gtfsvalidator/notice/GeojsonDuplicateKeyNotice.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 MobilityData
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mobilitydata.gtfsvalidator.notice;
+
+import static org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice.SectionRef.FILE_REQUIREMENTS;
+import static org.mobilitydata.gtfsvalidator.notice.SeverityLevel.ERROR;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice.SectionRefs;
+import org.mobilitydata.gtfsvalidator.table.GtfsLocationsSchema;
+
+/**
+ * Duplicated feature id.
+ *
+ * <p>One of the feature id of the locations.geojson file appears twice or more in the file.
+ */
+@GtfsValidationNotice(
+    severity = ERROR,
+    files = @GtfsValidationNotice.FileRefs(GtfsLocationsSchema.class),
+    sections = @SectionRefs(FILE_REQUIREMENTS))
+public class GeojsonDuplicateKeyNotice extends ValidationNotice {
```

Comment:
As discussed, we can create this notice more generic renaming it to `JsonDuplicateKeyNotice, adding it the field name, field value and removing references to the featureId.